packages/fuels/test/features/deploy.test.ts

```diff
@@ -106,6 +106,10 @@ describe('deploy', { timeout: 180000 }, () => {
       );
 
       const targetContract = new Contract(upgradableContractId, upgradableAbi, wallet);
+
+      const { value: getCountValue } = await targetContract.functions.get_value().get();
+      expect(getCountValue).toBe(10);
```

Comment:
These methods will revert without the correct storage slots, this is the only test we need.